src/runtime_src/core/tools/xbutil2/OO_MemWrite.cpp

```diff
@@ -205,6 +205,7 @@ OO_MemWrite::execute(const SubCmdOptions& _options) const
       addr += size;
     }
     std::cout << "Memory write succeeded" << std::endl;
+    return;
```

Comment:
Why is there a need to add `return` here?

Reply:
Without `return`, the code previously went on after a successful write to `printHelp()` and throws an error.